tests/unit/core/TestInstallAddon.js

```diff
@@ -601,6 +601,30 @@ describe(__filename, () => {
         .returns(false);
     });
 
+    describe('isAddonEnabled', () => {
+      it('calls addonManager.getAddon()', () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper();
+        let guid;
+
+        const addon = createInternalAddon({
+          ...fakeAddon,
+          _addonManager: fakeAddonManager,
+        });
+        const { root } = renderWithInstallHelpers({
+          ...addon,
+          addon: { guid },
+          _addonManager: fakeAddonManager,
+        });
+
+        guid = root.instance().props.guid;
```

Comment:
to me this seemed a little weird that I needed to do this but it doesn't work without this..?

Reply:
That does not sound good 😛

Reply:
lol..

actually I just noticed tho I think w/ the changes (I am not sure exactly what), its ok now without...?
I'll remove now...